api/models/templates/app.tmpl

```diff
@@ -251,6 +251,12 @@
         "Description" : "",
         "AllowedValues": [ "Yes", "No" ]
       },
+      "{{ upper $balancer.ProcessName }}Port{{ .Balancer }}InsecureProtocol": {
```

Comment:
can probably change InsecureProtocol to Protocol

Let's also make the default `TCP` so we remain backwards compatible.

Once you get that working run `make test` and see what breaks. You might need to update some of the CloudFormation outputs on tests.